nc/source/graphics/d3dresource/GraphicsResource.cpp

```diff
@@ -1,5 +1,6 @@
 #include "GraphicsResource.h"
 #include "graphics/Graphics.h"
+#include "graphics/Model.h"
```

Comment:
Why does GraphicsResource have to know about Model? That's pretty wonky we should redesign if this needs to be here